Client/Frontend/Browser/TabToolbar.swift

```diff
@@ -268,6 +272,11 @@ class TabToolbar: UIView {
         super.updateConstraints()
     }
 
+    override func layoutSubviews() {
+        super.layoutSubviews()
+        lineLayer.frame = CGRect(origin: .zero, size: CGSize(width: bounds.width, height: 1))
```

Comment:
it seems like the line height here is 1px where before it was 2px `setLineWidth(2)` ? Or is this a scale thing that I'm not understanding?